src/pages/troubleshooting/troubleshooting-timeouts.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Troubleshooting Timeouts"
+excerpt: "."
+categories: troubleshooting
+slug: timeouts
+toc: true
+---
+<!--                                                                    -->
+<!-- (C) Copyright Merative US L.P. and others 2018, 2023                -->
+<!--                                                                    -->
+<!-- SPDX-License-Identifier: Apache-2.0                                -->
+<!--                                                                    -->
+
+
+Timeouts can occur during analyze calls if the documents you send to ACD take too long to process.  This can be due to the size of the document or the configuration of the cartridge and its artifacts with the content of the document or a combination of both.  In addition there are various components in the path which can be configured to adjust their timeouts.  
```

Comment:
minor readability comment (comma):  In addition, there are various components in the path